src/main/java/net/mcreator/element/types/PotionEffect.java

```diff
@@ -45,6 +45,10 @@
 	public boolean renderStatusInInventory;
 	public boolean renderStatusInHUD;
 
+	public boolean isCuredByMilk;
```

Comment:
All three will be false here by default, while this is not default behavior so far for existing MEs. This needs to match current behavior or you will violate how converters should handle existing workspaces

Reply:
Should I start them at a value? (true for isCuredByMilk and isProtectedByTotem etc.)

Reply:
Do it how it is done with other MEs when correct values need to be initialized

Reply:
Ok

Reply:
Done